server/src/queries/getProfile.js

```diff
@@ -0,0 +1,9 @@
+const { User } = require('../models');
+
+exports.getProfile = (id) =>
+  User.findAll({
```

Comment:
Since there is one user for this id, use ```findOne``` to return the first object and then you don't need to destruct line 10 ;)
```suggestion
  User.findOne({
```